app/controllers/graphql_controller.rb

```diff
@@ -2,6 +2,7 @@
 
 class GraphqlController < ApplicationController
   before_action :fetch_params, only: [:execute]
+  skip_before_action :verify_authenticity_token
```

Comment:
The following should be added when we start doing adds, updates, and deletes:
`except: [:create, :update, :destroy]`